AdobeStockAsset/etc/db_schema.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
+    <table name="adobe_stock_request" resource="default" engine="innodb" comment="Adobe Stock License Request">
+        <column xsi:type="int" name="id" padding="10" unsigned="true" nullable="false" identity="true" comment="Entity ID"/>
+        <column xsi:type="int" name="admin_user_id" padding="10" unsigned="true" nullable="false" identity="false" default="0" comment="Admin User Id"/>
+        <column xsi:type="text" name="request_id" nullable="false" comment="Request ID"/>
+        <constraint xsi:type="primary" referenceId="PRIMARY">
+            <column name="id"/>
+        </constraint>
+        <constraint xsi:type="foreign" referenceId="ADO_STO_LIC_REQ_ADM_USER_ID" table="adobe_stock_license_request" column="admin_user_id" referenceTable="admin_user" referenceColumn="user_id" onDelete="CASCADE"/>
+    </table>
+    <table name="adobe_stock_user_profile" resource="default" engine="innodb" comment="Adobe Stock User Profile">
```

Comment:
please change table name from adobe_stock_user_profile to adobe_stock_profile

https://github.com/magento/adobe-stock-integration/wiki/Database#adobe-stock-user-profile-1

Reply:
I remember I discussed this with @sidolov 
I think the actual code is correct.